docker-compose.yml

```diff
@@ -1,12 +1,21 @@
 version: '3'
 services:
+
     web:
-        build: ./
-        container_name: svgtranslate_web
+        build:
+            dockerfile: ./.docker/web/Dockerfile
+            context: .
         ports:
-          - ${TOOLFORGE_DOCKER_PORT}:80
+            - ${TOOLFORGE_DOCKER_PORT}:80
         volumes:
-          - ./.docker/lighttpd.conf:/etc/lighttpd/lighttpd.conf:cached
-          - ./:/var/www:cached
+            - ./:/var/www:cached
         environment:
-          TOOLFORGE_DOCKER_PORT: ${TOOLFORGE_DOCKER_PORT}
+            TOOLFORGE_DOCKER_PORT: ${TOOLFORGE_DOCKER_PORT}
```

Comment:
Same thing here... do you need this in your application? I imagine your app should be able to just read the port of the request? 

If you would like to leave it, you should use [env_file](https://docs.docker.com/compose/compose-file/#env_file) rather than `environment`